### PR TITLE
Add automatic dependency on setTempRet0 for i64-returning functions. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/jsifier.js [hunk 1]
```diff
@@ -279,7 +286,7 @@ function ${name}(${args}) {
       }
 
       if (symbolsOnly) {
-        if (!isJsOnlySymbol(symbol) && LibraryManager.library.hasOwnProperty(symbol)) {
```
Comment (kripken) [inline src/jsifier.js:282]: Is this change related?

**Review by kripken — APPROVED** [review]
lgtm % question

File: src/jsifier.js (hunk 1 above)
Comment (sbc100) [reply src/jsifier.js:282]: Yup, this is because I removed the `INCLUDE_FULL_LIBRARY` hack in emcc.py.

Now we include JS symbols in the dependency info file too.    This way INCLUDE_FULL_LIBRARY can specify all possible native deps.. even native deps on JS symbols.

**Review by kripken — APPROVED** [review]
